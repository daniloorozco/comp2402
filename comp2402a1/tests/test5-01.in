cate
lines
should
be
be
be
once
take
special
care
so
that
a
file
with
a
lot
of
duplicate
lines
does
not
use
more
memory
than
what
is
required
for
the
number
of
unique
lines
