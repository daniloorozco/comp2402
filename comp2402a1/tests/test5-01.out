a
be
care
cate
does
duplicate
file
for
is
lines
lot
memory
more
not
number
of
once
required
should
so
special
take
than
that
the
unique
use
what
with
